src/account_processing.yml

```diff
@@ -172,7 +172,11 @@ Resources:
             Bucket:
               Ref: ADFAccountBucket
             Events: s3:ObjectCreated:*
-
+            Filter:
+              S3Key:
+                Rules:
+                - Name: suffix
+                  Value: yml
```

Comment:
We need to match `yaml` as well I think.

Reply:
Agreed.

As per: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-notificationconfiguration-config-filter-s3key.html

> The same type of filter rule cannot be used more than once. For example, you cannot specify two prefix rules.

This likely means I need to add another obj in the Events list as per: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-function-s3.html#sam-function-s3-events